---
apiVersion: "v1"
kind: "Event"
metadata:
  labels:
    EntandoResourceKind: "TestResource"
    EntandoResourceNamespace: "my-namespace"
    TestResource: "my-app"
  name: "my-app-42343806"
  namespace: "my-namespace"
  ownerReferences:
  - apiVersion: "test.org/v1"
    kind: "TestResource"
    blockOwnerDeletion: true
    controller: true
    name: "my-app"
    uid: "fa62f7cd-8f62-49c3-8685-028251c68dba"
action: "FAILED"
count: 1
firstTimestamp: "2021-06-22T07:14:00Z"
involvedObject:
  apiVersion: "test.org/v1"
  kind: "TestResource"
  fieldPath: "status"
  name: "my-app"
  namespace: "my-namespace"
  resourceVersion: "9316700"
  uid: "fa62f7cd-8f62-49c3-8685-028251c68dba"
lastTimestamp: "2021-06-22T07:14:00Z"
message: "The deployment of TestResource my-namespace/my-app failed due to Deployment\
  \ failed. Please inspect the logs of the pod my-namespace/my-app-deployment. Fix\
  \ the root cause and then trigger a redeployment by adding the annotation 'entando.org/processing-instruction:\
  \ force'"
reason: "Failed"
related:
  apiVersion: "v1"
  kind: "Pod"
  name: "test-controller-pod"
  namespace: "controller-namespace"
source:
  component: "entando-k8s-ource-controller"
type: "Error"
